integreat_cms/cms/templates/pages/page_pdf.html

```diff
@@ -84,7 +84,7 @@ <h1 id="title_page">{{ language.table_of_contents }}</h1>
             {% if page.depth < 3 %}
                 <h1 class="{% if page.depth == 1 %}root_title{% endif %}">{{ page_translation.title }}</h1>
             {% else %}
```

Comment:
```suggestion
            <h1 class="level-{{ page.depth }}">
                {{ page_translation.title }}
            </h1>
```
Maybe we could generalize this a bit and add a separate class for each level?